python/ray/tests/test_network_failure_e2e.py

```diff
@@ -0,0 +1,132 @@
+import sys
+import json
+
+from time import sleep
+import pytest
+from ray._private.test_utils import wait_for_condition
+from ray.tests.conftest_docker import *  # noqa
+from ray.tests.conftest_docker import gen_head_node, gen_worker_node
+
+
+SLEEP_TASK_SCRIPTS = """
+import ray
+ray.init(address="localhost:6379")
```

Comment:
nit: just `ray.init()` is fine, it will connect to the existing cluster.

Reply:
Fixed